community-operators/synopsys/synopsys-operator.2019.1.0.yaml

```diff
@@ -0,0 +1,288 @@
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/openshift-ds-ul.pdf
+   - name: Synopsys Operator Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/BDLM/pages/34373652/Synopsys+Operator
+   - name: Synopsys Operator Source Code
+     url: https://github.com/blackducksoftware/synopsys-operator
+  installModes:
+   - type: OwnNamespace
+     supported: false
+   - type: SingleNamespace
+     supported: false
+   - type: MultiNamespace
+     supported: false
+   - type: AllNamespaces
+     supported: true
+  install:
+    strategy: ReplicationController
+    spec:
+      replicationcontrollers:
+        spec:
+          replicas: 1
+          selector:
+            name: synopsys-operator
+          template:
+            metadata:
+              labels:
+                name: synopsys-operator
+            spec:
+              containers:
+              - args:
+                - /etc/synopsys-operator/config.json
+                command:
+                - ./operator
+                image: docker.io/blackducksoftware/synopsys-operator:2019.1.0
+                imagePullPolicy: Always
+                name: synopsys-operator
+                ports:
+                - containerPort: 8080
+                  protocol: TCP
+                resources: {}
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+                volumeMounts:
+                - mountPath: /etc/synopsys-operator
+                  name: synopsys-operator
+              - command:
+                - ./app
+                env:
+                - name: ADDR
+                  value: 0.0.0.0
+                - name: PORT
+                  value: "3000"
+                - name: GO_ENV
+                  value: development
+                image: docker.io/blackducksoftware/synopsys-operator:2019.1.0
+                imagePullPolicy: Always
+                name: synopsys-operator-ui
+                ports:
+                - containerPort: 3000
+                  protocol: TCP
+                resources: {}
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+              dnsPolicy: ClusterFirst
+              restartPolicy: Always
+              schedulerName: default-scheduler
+              securityContext: {}
+              serviceAccount: synopsys-operator
+              serviceAccountName: synopsys-operator
+              terminationGracePeriodSeconds: 30
+              volumes:
+              - configMap:
+                  defaultMode: 420
+                  name: synopsys-operator
+                name: synopsys-operator
+      permissions:
+      - serviceAccountName: synopsys-operator
+        rules:
+        - apiGroups:
+          - "*"
+          attributeRestrictions: null
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups: null
```

Comment:
For nonResourceUrls, no need to include null keys

Reply:
should have this fixed tomorrow morning

Reply:
should be fixed now